src/Packetery/Module/Carrier/OptionsPage.php

```diff
@@ -136,6 +137,45 @@ private function createForm( array $carrierData ): Form {
 		$form->addText( self::FORM_FIELD_NAME, __( 'Display name', 'packeta' ) . ':' )
 			->setRequired();
 
+		$availableVendors = [];
+		if ( strpos( $carrierData['id'], 'zpoint' ) === 0 ) {
+			$zPointCarriers = $this->carrierRepository->getZpointCarriers();
+			foreach ( $zPointCarriers as $zpointCarrier ) {
+				$availableVendors[ $zpointCarrier['id'] ] = $zpointCarrier['vendors'];
+			}
+		}
+
+		$carrierOptions = get_option( $optionId );
+		if (
+			! empty( $availableVendors ) &&
+			! empty( $availableVendors[ $carrierData['id'] ] )
+		) {
+			$vendors = $this->carrierRepository->getVendorCarriers();
+
+			if ( count( $availableVendors[ $carrierData['id'] ] ) <= self::MINIMUM_CHECKED_VENDORS ) {
+				$hiddenVendors = $form->addContainer( 'vendor_defaults' );
```

Comment:
vysvětli mi k čemu potřebuješ tento kontejner

Reply:
Disabled inputy se neodesílají, zajišťuje to, aby byly uloženy správné hodnoty bez toho, aby se muselo při ukládání zjišťovat, že (a jaké) mají být uloženy.

Reply:
Tak je nenastavuj jako disabled, ale dej jim readonly atribut. A tento kontejner smaž.